pkg/rehearse/rehearse.go

```diff
@@ -569,6 +576,15 @@ func setupDependencies(
 			if err := ensureImageStreamTags(ctx, client, requiredImageStreamTags, buildCluster, prowJobNamespace, prowJobClient, log); err != nil {
 				return fmt.Errorf("failed to ensure imagestreamtags in cluster %s: %w", buildCluster, err)
 			}
+			ocClientFactory := quayiociimagesdistributor.NewClientFactory()
+			quayIOImageHelper, err := ocClientFactory.NewClient()
```

Comment:
Could we put those two in main() and pass `quayIOImageHelper` into the function?
It should be reusable.